syntax_checkers/twig/twigcs.vim

```diff
@@ -0,0 +1,41 @@
+"============================================================================
+"File:        twigcs.vim
+"Description: Syntax checking plugin for syntastic
+"Maintainer:  Ciloe <escrichjimmy at yahoo dot fr>
+"License:     This program is free software. It comes without any warranty,
+"             to the extent permitted by applicable law. You can redistribute
+"             it and/or modify it under the terms of the Do What The Fuck You
+"             Want To Public License, Version 2, as published by Sam Hocevar.
+"             See http://sam.zoy.org/wtfpl/COPYING for more details.
+"
+"============================================================================
+
+if exists('g:loaded_syntastic_twig_twigcs_checker')
+    finish
+endif
+let g:loaded_syntastic_twig_twigcs_checker = 1
+
+let s:save_cpo = &cpo
+set cpo&vim
+
+function! SyntaxCheckers_twig_twigcs_GetLocList() dict
+    let makeprg = self.makeprgBuild({ 'args_after': '--reporter=emacs' })
+    " example report line : path/to/file/a.html.twig:1:8: warning - Unused variable "foo".
+    " available report values : ignore, info, warning, error
+    let errorformat = '%f:%l:%c: %t - %m'
```

Comment:
`%t` in `errorformat` matches a single character, not something like `warning`.

Reply:
I can't see a documentation about the error format parameters. What can I use to match all report values possibilities ?

Reply:
`:h errorformat`

Reply:
Like in phpCS I think I can `%f:%l:%c: %t%*[a-z] - %m`